src/Templates/src/templates/maui-mobile/Resources/Styles/Styles.xaml

```diff
@@ -343,21 +343,19 @@
         <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource White}, Dark={StaticResource Black}}" />
     </Style>
 
-    <Style x:Key="BaseStyle" TargetType="Element">
+    <Style TargetType="Shell" ApplyToDerivedTypes="True">
         <Setter Property="Shell.BackgroundColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource Black}}" />
-        <Setter Property="Shell.ForegroundColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}" />
+        <Setter Property="Shell.ForegroundColor" Value="{OnPlatform WinUI={StaticResource Primary}, Default={x:Null}}"></Setter>
```

Comment:
Ideally we could specify different AppThemeBindings for each platform but that doesn't seem possible at this point

https://github.com/xamarin/Xamarin.Forms/issues/11662